go/erasablekv/erasable_kv_store.go

```diff
@@ -86,22 +101,28 @@ func (s *FileErasableKVStore) unbox(ctx context.Context, data []byte, noiseBytes
 	// Decode encrypted box
 	var boxed boxedData
 	if err := libkb.MPackDecode(data, &boxed); err != nil {
-		return err
+		return NewUnboxError(err.Error())
```

Comment:
Do you want to just keep the underlying error around, instead of just the string?

Reply:
yes better